### ISSUE TITLE
CLI setup

### ISSUE THREAD

**Issue by nicolashefti** [formulajs/formulajs#98]
Hi @bornova 

I just pulled the last master version to test the cli.

1) I followed the README. After running `npm i -g @formulajs/formulajs` I don't have any global formulajs. I checked in the global executable folder (where `ng` or `npx`live) but nothing here regarding `formulajs`.

2) In the repo, after installation.

```sh
./bin/cli.js   # => permission denied: ./bin/cli.js

chmod +x bin/cli.js

./bin/cli.js  # => [ERR_MODULE_NOT_FOUND]: Cannot find module '.../index.mjs' imported from '.../bin/cli.js'
```

Can you clarify how this should work ?

**Comment by bornova** [comment]
Hi @nicolashefti ,
A new build needs to be released for the CLI to work.  Currently, the newly added cli.js doesn't exist in the current 3.0.0 build and package.json doesn't point to the new cli file.  Once a new version is released, it should work as described in README.md

You can also build locally and test with `npm link` if you'd like before releasing a new version.

**Comment by nicolashefti** [comment]
I'm not used to create such executable package. Thanks for the explanation.

Just released version 3.1.1. Works perfectly!

**Comment by nicolashefti** [comment]
@bornova in case your on Twitter, feel free to spread the news 👍  https://twitter.com/Appizy/status/1497325092505849865
